### PR TITLE
Update list of known network locations

### PR DESCRIPTION
**Pull request by rickettmwork**
- [x] I have considered the risk of behavior change or other zero migration impact (https://github.com/OpenLiberty/open-liberty/wiki/Behavior-Changes).
- [x] If this PR fixes an Issue, the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN" (verify `release bug` label if applicable: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions).
- [x] If this PR resolves an external Known Issue (including APARS), the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN".

### CODE REVIEW

**Comment by rickettmwork** [comment]
!build ([view Open Liberty Personal Build](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=e4238ff1-ece3-473c-a4b1-e7e4ab9cf293) - :x: completed with errors/failures.)

_Note: Target locations of links might be accessible only to IBM employees._

**Review by websterc87 — DISMISSED** [review]
Approved subject to a passing PB.

**Comment by rickettmwork** [comment]
!build ([view Open Liberty Personal Build](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=3728b241-d2d7-4652-b9a1-eb2337a35ecf) - :x: completed with errors/failures.)

_Note: Target locations of links might be accessible only to IBM employees._

File: dev/fattest.simplicity/src/componenttest/topology/utils/ExternalTestService.java [hunk 1]
```diff
@@ -604,14 +604,29 @@ private static String getNetworkLocation() {
...
+	    // POK
+	    return "IBM9US";
+        } else if (serverOrigin.startsWith("10.34.") || serverOrigin.startsWith("10.36.")) {	    
+            return "HURPROD";
+	} else if (serverOrigin.startsWith("10.51.")) {
+	    return "FYREHUR";
+	} else if (serverOrigin.startsWith("10.17.") || serverOrigin.startsWith("10.11.") || serverOrigin.startsWith("10.15.")) {
+	    return "FYRESVL";
+	} else if (serverOrigin.startsWith("10.21.") || serverOrigin.startsWith("10.21.")) {
+	    return "FYRERTP";
```
Comment (KyleAure) [inline dev/fattest.simplicity/src/componenttest/topology/utils/ExternalTestService.java]: This line looks like it has a typo did you intend to repeat a check for `10.21.`?

**Review by KyleAure — DISMISSED** [review]
Approved, because the code will work, but left a comment for a potential typo.

**Review by KyleAure — APPROVED** [review]

**Comment by rickettmwork** [comment]
!build
 ([view Open Liberty Personal Build](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=effa87b9-a96d-4a8f-a126-12e4d7a24a22) - :x: completed with errors/failures.)

_Note: Target locations of links might be accessible only to IBM employees._

**Comment by rickettmwork** [comment]
PB failures are in z/os fats that are not caused by this change

**Pull request merged**